packages/components/src/templates/next/components/internal/Notification/Notification.tsx

```diff
@@ -14,19 +17,48 @@ const NotificationBanner = ({ content }: NotificationProps) => {
 
   return (
     isShown && (
-      <div className="flex flex-col justify-between gap-6 bg-site-primary-100 px-5 py-4 sm:flex-row sm:px-10">
-        <BaseParagraph
-          className="text-base text-content"
-          content={content}
-        ></BaseParagraph>
-        <button
-          className="ml-auto flex flex-row items-center gap-1"
-          aria-label="Dismiss notification"
-          onClick={onDismiss}
-        >
-          <p className="text-lg text-content">Dismiss</p>
-          <BiX className="h-full" />
-        </button>
+      <div className="bg-base-canvas-backdrop">
+        <div className="relative mx-auto flex max-w-screen-xl flex-row gap-4 px-6 py-8 text-base-content md:px-10 md:py-6">
+          <BiInfoCircle className="h-6 w-6 shrink-0" />
+          <div className="flex-1">
+            <Markdown
+              options={{
+                // slugify to ensure unique ids for headings
+                slugify: (input) => `notification-${input}`,
+                overrides: {
+                  a: {
+                    component: ({ children, ...props }) => {
+                      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+                      const isExternal = isExternalUrl(String(props.href ?? ""))
+                      return (
+                        <Link
+                          {...props}
+                          withFocusVisibleHighlight
+                          isExternal={isExternal}
+                          showExternalIcon={isExternal}
+                        >
+                          {children}
+                        </Link>
+                      )
+                    },
+                  },
+                  h2: {
+                    props: {
+                      className: "prose-headline-lg-medium",
+                    },
+                  },
+                },
+              }}
+            >{`${content}`}</Markdown>
+          </div>
+          <div aria-hidden className="flex h-6 w-6 shrink-0" />
+          <IconButton
+            onPress={onDismiss}
```

Comment:
question(non-blocking): curious is it possible to have the dismiss persist only during the session? So it stays dismissed while the user navigates around the site, but it re-appears the next time they open the site again.

Maybe it already comes for free now since we are using next/link but not sure.

Reply:
doesn't come for free. can probably code it in using session token.